point_viewer_grpc/src/bin/octree_benchmark.rs

```diff
@@ -71,14 +72,20 @@ fn server_benchmark(octree_directory: &Path, num_points: u64) {
         )
     });
     let mut counter: u64 = 0;
-    octree.all_points().for_each(|_p: Point| {
-        if counter % 1_000_000 == 0 {
-            println!("Streamed {}M points", counter / 1_000_000);
-        }
-        counter += 1;
-        if counter == num_points {
+    let all_points = PointQuery {
+        location: PointLocation::AllPoints(),
+        global_from_local: None,
+    };
+    let mut batch_iterator = BatchIterator::new(&octree, &all_points, 1_000_000);
+    let mut num_batches = 0;
```

Comment:
Nit: How about having a const variable `BATCH_SIZE`, removing `num_batches`, and just working on `counter`?

Reply:
done

Reply:
done